### PR TITLE
Refactor/clean samples code

### PR DESCRIPTION
**Pull request by fchauveau**
Refactor on picker builder to have fluent setters and all same way to instantiate a picker
- refactor on samples (extract string resources / clean some code)

### CODE REVIEW

File: library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java [hunk 1]
```diff
@@ -144,22 +144,7 @@
...
-        if (dark) {
-            mStyleResId = R.style.BetterPickersRadialTimePickerDialog_Dark;
-        } else {
-            mStyleResId = R.style.BetterPickersRadialTimePickerDialog;
-        }
-    }
-
-    public void setThemeCustom(int styleResId) {
-        this.mStyleResId = styleResId;
+        mStyleResId = R.style.BetterPickersRadialTimePickerDialog;
```
Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java:147]: This looks like a no-setter : we always set the value taken from the theme... Am I missing something ?

**Comment by CedricGatay** [comment]
Except for the comment that needs to be checked, it looks good to me.
Good work !

File: library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java (hunk 1 above)
Comment (fchauveau) [inline library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java:147]: "mStyleResId = R.style.BetterPickersRadialTimePickerDialog;"
Is done in constructor to provide a default theme, that can be override using setter in line 180 and 189

Comment (CedricGatay) [inline library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java:147]: My bad, I've misread the diff.

**Pull request merged**